NEWS

```diff
@@ -17,14 +17,14 @@ support for Python 2.6 is considered to be deprecated. It has also been
 tested on PyPy 2.4 to 2.6, PyPy3 version 2.4, and Jython 2.7.
 
 Comparison of SeqRecord objects until now has used the default Python object
-comparison (are they same instance in memory?). This can be surprising, but
+comparison (are they the same instance in memory?). This can be surprising, but
 comparing all of the attributes would be too complex. As of this release
 attempting to compare SeqRecord objects should raise an exception instead. If
 you want the old behaviour, use id(record1) == id(record2) instead.
 
 Following the convention used elsewhere in Biopython, there is a new function
 Bio.KEGG.read(...) for parsing KEGG files expected to contain a single record
-only - existing function Bio.KEGG.parse(...) is intended to be used to iterate
+only - the existing function Bio.KEGG.parse(...) is intended to be used to iterate
```

Comment:
Adding "the" is fine, but line wrap before iterate (we try to keep to 78 chars in this file for old school terminal usage).